### PR TITLE
Extend preview notice to .NET 8 doc versions

### PR DESCRIPTION
**Pull request by tdykstra**
---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/fundamentals/error-handling.md](https://github.com/dotnet/AspNetCore.Docs/blob/f313a20b253ae1cd0e6b287b4b7f0e5a4ed2f84e/aspnetcore/fundamentals/error-handling.md) | [Handle errors in ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/error-handling?branch=pr-en-us-29844) |
| [aspnetcore/fundamentals/servers/kestrel/endpoints.md](https://github.com/dotnet/AspNetCore.Docs/blob/f313a20b253ae1cd0e6b287b4b7f0e5a4ed2f84e/aspnetcore/fundamentals/servers/kestrel/endpoints.md) | [Configure endpoints for the ASP.NET Core Kestrel web server](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel/endpoints?branch=pr-en-us-29844) |
| [aspnetcore/mvc/controllers/routing.md](https://github.com/dotnet/AspNetCore.Docs/blob/f313a20b253ae1cd0e6b287b4b7f0e5a4ed2f84e/aspnetcore/mvc/controllers/routing.md) | [Routing to controller actions in ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/mvc/controllers/routing?branch=pr-en-us-29844) |

### CODE REVIEW

File: aspnetcore/fundamentals/minimal-apis/handle-errrors.md [hunk 1]
```diff
@@ -8,9 +8,7 @@ ms.date: 5/2/2023
 uid: fundamentals/minimal-apis/handle-errors
 ---
 
-<!-- Can't add this until 8 is released and not-latest-version.md is updated to 8.0
 [!INCLUDE[](~/includes/not-latest-version.md)]
--->
```
Comment (tdykstra) [inline aspnetcore/fundamentals/minimal-apis/handle-errrors.md:12]: This article now has 8.0 content, and while 6 and 7 versions are identical, if 6 or anything earlier is selected it automatically switches to 7, so you'll never get the "not latest" warning.

File: aspnetcore/mvc/controllers/routing.md [hunk 2]
```diff
@@ -10,9 +10,7 @@ uid: mvc/controllers/routing
 
 By [Ryan Nowak](https://github.com/rynowak), [Kirk Larkin](https://twitter.com/serpent5), and [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-:::moniker range="< aspnetcore-6.0"
-[!INCLUDE[](~/includes/not-latest-versionNM.md)]
-:::moniker-end
+[!INCLUDE[](~/includes/not-latest-version.md)]
```
Comment (tdykstra) [inline aspnetcore/mvc/controllers/routing.md:13]: 6 and 7 are identical, so it isn't necessary to flag 6 as old, but it doesn't hurt to direct people to 7 for the latest version.

File: aspnetcore/fundamentals/minimal-apis/handle-errrors.md [hunk 3]
```diff
@@ -4,15 +4,12 @@ author: brunolins16
 description: Learn about error handling with minimal APIs in ASP.NET Core.
 ms.author: brolivei
 monikerRange: '>= aspnetcore-7.0'
-ms.date: 5/2/2023
+ms.date: 07/18/2023
 uid: fundamentals/minimal-apis/handle-errors
 ---
+# How to handle errors in Minimal API apps
```
Comment (tdykstra) [inline aspnetcore/fundamentals/minimal-apis/handle-errrors.md:10]:
```suggestion
```

File: aspnetcore/fundamentals/minimal-apis/handle-errrors.md [hunk 4]
```diff
@@ -4,15 +4,12 @@ author: brunolins16
...
 
-<!-- Can't add this until 8 is released and not-latest-version.md is updated to 8.0
 [!INCLUDE[](~/includes/not-latest-version.md)]
```
Comment (tdykstra) [inline aspnetcore/fundamentals/minimal-apis/handle-errrors.md:12]:
```suggestion
<!-- Can't add this until 8 is released and not-latest-version.md is updated to 8.0
[!INCLUDE[](~/includes/not-latest-version.md)]
--> 

# How to handle errors in Minimal API apps
```

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**